app/src/main/java/doubleslash05/mini/team11/ui/tutorial/TutorialActivity.kt

```diff
@@ -0,0 +1,157 @@
+package doubleslash05.mini.team11.ui.tutorial
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import androidx.viewpager.widget.ViewPager
+import com.google.android.material.tabs.TabLayout
+import com.icaksama.rapidsphinx.RapidPreparationListener
+import com.icaksama.rapidsphinx.RapidSphinx
+import com.icaksama.rapidsphinx.RapidSphinxListener
+import doubleslash05.mini.team11.R
+import edu.cmu.pocketsphinx.Config
+
+class TutorialActivity : AppCompatActivity(), RapidSphinxListener {
+    private val stopKeyword = "stop"
+    private val nextKeyword = "next"
+    private val pauseKeyword = "pause"
+    private var vpPager: ViewPager? = null
+    private var customTab: TabLayout? = null
+    private var rapidSphinx: RapidSphinx? = null
```

Comment:
nullable 을 사용하는 것은 좋은 습관이 아닙니다.
lateinit 을 사용 하실 수도 있지만 
by lazy 에 대해서도 공부해 보세요.

```suggestion
    private val rapidSphinx: RapidSphinx by lazy { RapidSphinx(this) }
```

특별한 사유가 없으면
접근제한자는 private, 변수 선언은 val 로 습관을 들이시는 것이 좋습니다.